ui/src/components/NavigationBar/MainMenu.js

```diff
@@ -0,0 +1,456 @@
+import React, { useState } from "react";
+import { Link } from "react-router-dom";
+import {
+  Flex,
+  Box,
+  HStack,
+  Button,
+  ButtonGroup,
+  VStack,
+  Text,
+  Center,
+  Accordion,
+  AccordionItem,
+  AccordionButton,
+  AccordionPanel,
+  Container,
+  AccordionIcon,
+} from "@chakra-ui/react";
+import { SocialIcon } from "react-social-icons";
+
+const MainMenu = ({ login, view }) => {
+  const WebView = () => {
+    return (
+      <Container maxW="container.lg">
+        <HStack display={{ base: "none", md: "flex" }} color="white">
+          <Flex marginTop="30px">
+            <VStack alignItems="flex-start">
+              <Text fontWeight="bold" fontSize="40px">
+                Visit
+              </Text>
+              <Link
+                to={{ pathname: "https://hongkong.hmcc.net/online/" }}
```

Comment:
this is not the correct way to do an outlink. for outlinks you should be doing a `href`. Using a `to` is for local pathnames. If you do this, this will append the pathname that you specify here to the localhost so it would link to `localhost/https://hongkong.hmcc.net/online`, which is incorrectly done.